src/components/draggable/draggable.pw.tsx

```diff
@@ -92,11 +96,11 @@ test("calls container's getOrder callback with the correct arguments when an ite
     .toEqual([["mercury", "venus", "apple"], "apple"]);
 });
 
-test("calls fruit container's getOrder callback with correct arguments when a planet item is attempted to be dragged and dropped into the fruit container", async ({
+test("does not call fruit container's getOrder callback with correct arguments when a planet item is attempted to be dragged and dropped into the fruit container", async ({
```

Comment:
Noticed this was a mistake, given `DraggableContainer`'s `getOrder` prop should only be called when items have been reordered